src/wsiToDcm.h

```diff
@@ -22,51 +22,40 @@
 
 namespace wsiToDicomConverter {
 
+// Structure for wsi2dcm settings
+struct WsiRequest {
+  std::string inputFile = "";  // wsi input
+  std::string outputFileMask = "./";  // path to save generated files
+  int64_t frameSizeX = 500;
+  int64_t frameSizeY = 500;
+  DCM_Compression compression = JPEG;  // compression - jpeg, jpeg2000, raw
+  int32_t quality = 80;
+  int32_t startOnLevel = 0;   //
+  int32_t stopOnLevel = -1;   // limitation which levels to generate
+  std::string imageName = "image";  //
```

Comment:
Describe this parameter... is this set as part of the DICOM metadata? If so, which tag?

Reply:
done